### PR TITLE
feat: static generated library

### PR DESCRIPTION
**Pull request by Duologic**
This generates another static version of this library following the pattern I've used successfully in other libraries.

It doesn't include the `reusableWorkflowCallJob` part, not sure yet what that is used for. I'm new to GitHub Actions.

The generator depends on https://github.com/crdsonnet/crdsonnet/pull/24

### CODE REVIEW

File: main.libsonnet [hunk 1]
```diff
@@ -1,29 +1,7 @@
...
-      },
-    },
-  } },
-  'normalJob.libsonnet': unwrapFromCRDsonnet(crdsonnet.schema.render('normalJob', jobSchema, processor), 'normalJob').toString(break='\n'),
-  'reusableWorkflowCallJob.libsonnet': unwrapFromCRDsonnet(crdsonnet.schema.render('reusableWorkflowCallJob', schema.definitions.reusableWorkflowCallJob, processor), 'reusableWorkflowCallJob').toString(break='\n'),
+(import './raw.libsonnet')
++ {
+  job+: {
+    steps:: {},
+    step: (super.steps + { '#'+: { name: 'step' } }),
```
Comment (Duologic) [inline main.libsonnet:5]: I just noticed that this doesn't work with the language server. :shrug:

Comment (Duologic) [reply main.libsonnet:5]: Started a PR with testcases for this here: https://github.com/grafana/jsonnet-language-server/pull/144

**Comment by julienduchesne** [comment]
Have you found a use for jsonnet -> github actions workflows? I have made some fairly complex workflows in GHA and haven't found the need yet. The matrix feature is pretty powerful. Drone needed it because the matrix stuff was terrible

**Review by julienduchesne — APPROVED** [review]
:shipit:

**Pull request merged**